curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff7f.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 683ec95f2f8781355556ff7f
+title: Step 7
+challengeType: 20
+dashedName: step-7
+---
+
+# --description--
+
+Now, include `value_at_middle` in the `path_to_target` list to track the steps taken during the search, regardless of whether it's the target.
+
+# --hints--
+
+You should use the `append()` method to add `value_at_middle` to the `path_to_target` list. Don't forget to maintain the same indentation level as the existing variables in the while loop.
```

Comment:
same here 

I question if this note is needed here about the indentation level.
by this point in the curriculum, campers should understand the importance of indentation in python and be able to resolve indentation errors on their own.

IMO, it should be removed